ui/packages/catalog-ui-search/src/main/webapp/component/dropdown/query-src/dropdown.query-src.view.js

```diff
@@ -17,13 +17,67 @@ const Marionette = require('marionette')
 const _ = require('underscore')
 const $ = require('jquery')
 const DropdownView = require('../dropdown.view')
-const template = require('./dropdown.query-src.hbs')
-const ComponentView = require('../../query-src/query-src.view.js')
-const sources = require('../../singletons/sources-instance.js')
-const properties = require('../../../js/properties.js')
+const ComponentView = require('component/query-src/query-src.view')
+const sources = require('component/singletons/sources-instance')
+const properties = require('properties')
+import * as React from 'react'
+import { IntlProvider, FormattedMessage } from 'react-intl'
 
 module.exports = DropdownView.extend({
-  template: template,
+  template(props) {
+    return (
+      <div>
+        <div className="dropdown-label">
+          <IntlProvider locale="en">
+            <FormattedMessage
+              id="sourcesKeyword"
+              defaultMessage={`{sources}`}
+              values={{ sources: properties.keywords['Sources'] }}
+            />
+          </IntlProvider>
+        </div>
+        <div className="dropdown-container">
+          <div className="dropdown-text is-input">
+            {props.enterprise ? (
+              <span className="text-src is-available">
+                <IntlProvider locale="en">
+                  <FormattedMessage
+                    id="sourcesKeyword"
+                    defaultMessage={`All {sources}`}
+                    values={{ sources: properties.keywords['Sources'] }}
+                  />
+                </IntlProvider>
+              </span>
+            ) : (
+              props.sources.map(source => (
```

Comment:
✏️ Might be a little more readable to pull this up into `renderSourceAvailable = (source) => <>...</>` and `renderSourceLocal = (source) => <>...</>`